### ISSUE TITLE
FGS-221: Missing status updates in Homey

### ISSUE THREAD

**Issue by JPe2** [athombv/homey#2913]
**Metadata**

* **Homey Version**:Homey Firmware Versie: 1.5.13-rc.11
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**Fibaro 
* **Homey App Version**: v1.5.18
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**: FGS-221

**What did you try to achieve?**

When the inputs of the device FGS-221 (S1 / S2) are switched by the connected switches, the flows that should be triggered, do not run and also the Icons are not changing to ON or OFF
I made flows for both switches and for both states ( On and OFF )

**What did you expect as result?**

Expecting (surprise) that the flows would start and that the status of the Icons would change.

**What was the result?**

The result is that the flows do not start and the Icons do not change status.

**Any other remarks**
As far as I can see, the messages, indicating the status change, are being sent:
2018-11-06T16:35:23.726Z
Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x0300
2018-11-06T16:35:23.753Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0101200300
2018-11-06T16:35:23.782Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d02022003ff
2018-11-06T16:35:29.557Z
Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x0300
2018-11-06T16:35:29.575Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0101200300
2018-11-06T16:35:29.594Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300
2018-11-06T16:35:34.319Z
Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x03ff
2018-11-06T16:35:34.364Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d01012003ff
2018-11-06T16:35:34.378Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300
2018-11-06T16:35:34.514Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300
2018-11-06T16:35:43.458Z
Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x0300
2018-11-06T16:35:43.476Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0101200300
2018-11-06T16:35:43.496Z
Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300

FGS-221
Node ID          49
Fabr. ID          271
Prod.TypeID  512
Prod.ID          260

**Comment by caseda** [comment]
Did you change any of the device's settings? in particular the association groups, as COMMAND_CLASS_BASIC is normally not the signal that is send to homey (for this device) but other association grouped devices.
it should have been COMMAND_CLASS_SWITCH_BINARY

**Comment by caseda** [comment]
what are your association settings then? the thing you showed last doesn't tell anything homey uses.

it should be homey's ID (1) in only association group 3.

**Comment by JPe2** [comment]
Thanks @caseda for your response, the association settings where 1 -- 1  and   3 -- 1
changed to  
![image](https://user-images.githubusercontent.com/3833862/48280266-67d17980-e453-11e8-98da-a78eeb93e189.png)
but no change in the log and no flows starting

2018-11-09T18:07:29.894Z | Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x03ff
-- | --
2018-11-09T18:07:29.912Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d01012003ff
2018-11-09T18:07:29.935Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300
2018-11-09T18:07:34.165Z | Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x03ff
2018-11-09T18:07:34.185Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d01012003ff
2018-11-09T18:07:34.225Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d02022003ff
2018-11-09T18:07:38.305Z | Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x0300
2018-11-09T18:07:38.325Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0101200300
2018-11-09T18:07:38.343Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d02022003ff
2018-11-09T18:07:40.446Z | Node[49]: Received application command for COMMAND_CLASS_BASIC, data: 0x0300
2018-11-09T18:07:40.473Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0101200300
2018-11-09T18:07:40.500Z | Node[49]: Received application command for COMMAND_CLASS_MULTI_CHANNEL, data: 0x0d0202200300

**Comment by caseda** [comment]
no, group 1 = 1 is only for _z-wave plus_ devices (99% of the time),

the fgs-221 is still a _normal z-wave_ device, where it is still fully manufacturer specific so you should have to look it up in the manual

**Comment by JPe2** [comment]
Thanks for this Z-Wave course :) .This is what I found 
> Fibaro Switch provides association of three groups.
> 1st group is assigned to key no. 1
> 2nd group is assigned to key no. 2
> 3rd group reports state of devices. Only one device can be
> associated to this group.
> Fibaro Switch enables user to control up to 16 normal devices, and 7
> MultiChannel devices in group no. 1 and no. 2, group no. 3 has only
> one field. _The first field in each group is reserved for the network_
> _controller_.
The last line may be the reason that the working doesn't change with different settings?

**Comment by canedje** [comment]
I did report this issue a year ago.
I do have the same problem still
